contents/handbook/strategy/strategy.md

```diff
@@ -72,6 +72,11 @@ While there is value in the items on the right, we value the items on the left m
 
 * _Self-hosted_: focus on customers who need to self-host, as there is a huge untapped market and we’re uniquely placed to win.
 
+* Why maintain Cloud if the focus is on Self-Hosted?
+1. **Top of Funnel:** Cloud acts as a top-of-funnel for people to quickly try out PostHog before committing resources to self-hosting
+2. **Iterate Fast:** Cloud enables us to quickly release, test and iterate on new features with our customers without waiting for longer self-hosting release cycles
+3. **Benchmark for scale:** If we can support a very large instance of PostHog then we can be confident that our customers can support an instance of similar size
+
```

Comment:
```suggestion
4. **Escape valve for users that made the wrong choice** Some teams think self hosting is what they want, but realize that they'd rather migrate to cloud once they're already in production and like the product, but want zero maintenance.
```

Maybe this isn't ambitious enough (ie management should be trivial) - but it is the case today.

Reply:
To be honest, this sounds more like a tactical reason rather than a strategic one.

Our strategic hypothesis is large enterprises need self-hosting, so this suggests that if they're probably not in our target audience.

However I think its worth including as context:
```suggestion
4. **Tactical: Escape valve for users that made the wrong choice** Some teams think self hosting is what they want, but realize that they'd rather migrate to cloud once they're already in production and like the product, but want zero maintenance.
```